sciplot/Plot.hpp

```diff
@@ -900,10 +900,21 @@ inline auto Plot::repr() const -> std::string
     script << "#==============================================================================" << std::endl;
     script << "plot \\\n"; // use `\` to have a plot command in each individual line!
 
+    // Use either regular or custom plot commands
+    const auto & numCmds = m_usecustomdrawcmds ? m_customdrawcmds.size() : m_drawspecs.size();
+
     // Write plot commands and style per plot
-    const auto n = m_drawspecs.size();
-    for(std::size_t i = 0; i < n; ++i)
-        script << "    " << m_drawspecs[i] << (i < n - 1 ? ", \\\n" : ""); // consider indentation with 4 spaces!
```

Comment:
We should definitely have these lines back. And these would be sufficient to deal with both specific plot commands and the custom one.